freq/lizard_api_connector.py

```diff
@@ -298,4 +306,17 @@ class GroundwaterLocations(Locations):
     Makes a connection to the locations endpoint of the lizard api.
     Only selects GroundwaterStations.
     """
-    extra_queries = {"object_type\__model": "GroundwaterStation"}
+    extra_queries = {
+        "object_type\__model": "GroundwaterStation",
+        "organisation__unique_id":"f757d2eb6f4841b1a92d57d7e72f450c"
```

Comment:
- Spatie achter `:`
- Klopt dat dit een hardcoded ID is?
- Organisatie ID uit secrets halen?